packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -43,8 +130,73 @@ const ReleasesPage = () => {
           </CheckPermissions>
         }
       />
+      <ContentLayout>
+        <>
+          <TabGroup
+            label={formatMessage({
+              id: 'content-releases.pages.Releases.tab-group.label',
+              defaultMessage: 'Releases list',
+            })}
+            variant="simple"
+            initialSelectedTabIndex={['pending', 'done'].indexOf(activeTab)}
+            onTabChange={handleTabChange}
+          >
+            <Tabs>
+              <Tab>
+                {formatMessage({
+                  id: 'content-releases.pages.Releases.tab.pending',
+                  defaultMessage: 'Pending',
+                })}
+              </Tab>
+              <Tab>
+                {formatMessage({
+                  id: 'content-releases.pages.Releases.tab.done',
+                  defaultMessage: 'Done',
+                })}
+              </Tab>
+            </Tabs>
+            <TabPanels>
+              {/* Pending releases */}
+              <TabPanel>
+                {isError ? (
```

Comment:
we don't have a mockup of the error page, in my mind it was to show the AnErrorOccured content in every single tab panel and show "No releases" as subtitle (or we can remove the subtitle), because the error can occur in every single tab (every tab call the api with different params).